### DISCUSSION TITLE
How can I change the mouse behavior of `ScrollView`?

### DISCUSSION THREAD

**Discussion by Poker-sang** [microsoft/microsoft-ui-xaml#9363] [Q&A]
When I scroll the wheel, I want to zoom the `ScrollView`. but when I use the `PointerWheelChanged` event with `ZoomTo` or `AddZoomVelocity`, the content always jerks. What should I do to solve this?

**Comment by RBrid** [comment]
Maybe setting ScrollView.IgnoredInputKinds to ScrollingInputKinds.MouseWheel will help.
(https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.scrollview.ignoredinputkinds?view=windows-app-sdk-1.4)

**Reply by Poker-sang** [reply]
Yes. But how can I zoom it fluently? ZoomTo or AddZoomVelocity jerks.
